packages/opc-footer/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "@one-platform/opc-footer",
+  "version": "0.0.1",
```

How about a pre-release tag for all the components? That will denote that the components are not completely ready/fully functional.

```suggestion
  "version": "0.0.1-prerelease",
```